### PR TITLE
#trivial [Docs] Improve the docstring for inherit!

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

**Comment by k0nserv** [comment]
I can add an explanation for each mode too, but I am not 100% sure on `complete`. I guess it means the target inherits pods, search paths and other settings from the outer target?

**Comment by k0nserv** [comment]
@orta

**Comment by orta** [comment]
I didn't know `:complete` and `none` existed, I assume `:complete` is the default ( get all target's stuff behaviour ) and none is a "this doesn't connect at all" (maybe for extensions?)

**Comment by k0nserv** [comment]
@segiddins would know I guess?

**Comment by segiddins** [comment]
@orta nailed it

File: lib/cocoapods-core/podfile/dsl.rb [hunk 1]
```diff
@@ -397,9 +397,18 @@ def abstract!(abstract = true)
...
       #
-      # @param   [:complete, :none, :search_paths] inheritance
+      # @param   [Symbol] inheritance
       #          the inheritance mode to set.
       #
+      #          **Available Modes:**
+      #          + `:complete` The target inherits all
+      #          behavior from the parent.
+      #          + `:none` The target inherits none of
+      #          the behaviour from the parent.
```
Comment (k0nserv) [inline lib/cocoapods-core/podfile/dsl.rb]: re: behaviour. British or America english for these docs?

Comment (segiddins) [inline lib/cocoapods-core/podfile/dsl.rb]: British

Comment (k0nserv) [inline lib/cocoapods-core/podfile/dsl.rb]: 👍 🇬🇧

**Comment by k0nserv** [comment]
Okay this is good to go then

File: lib/cocoapods-core/podfile/dsl.rb [hunk 2]
```diff
@@ -397,9 +397,18 @@ def abstract!(abstract = true)
 
       # Sets the inheritance mode for the current target.
       #
-      # @param   [:complete, :none, :search_paths] inheritance
+      # @param   [Symbol] inheritance
       #          the inheritance mode to set.
       #
+      #          **Available Modes:**
+      #          + `:complete` The target inherits all
+      #          behavior from the parent.
```
Comment (DanToml) [inline lib/cocoapods-core/podfile/dsl.rb]: /s/behavior/behaviour

Comment (k0nserv) [inline lib/cocoapods-core/podfile/dsl.rb]: Lol at the fact that I mixed them 😂

**Pull request merged**